base_locale_uom_default/tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
please remove this line. no longer needed

Reply:
you missed that one @skukered

Reply:
Sorry... Done